doc/Video Receiver.md

```diff
@@ -0,0 +1,53 @@
+# Video Receiver
+
+RotorHazard supports wireless communications with video receivers. Initial support is released for the ClearView2.0 receiver. The following capabalities are added with this functionality:
+
+* Race Manager can change frequencies of timer node and video receivers at the same time to avoid frequency conflicts
+* Race Manager can view if video receivers are locked onto a video signal
+* Pilots can see race start and finish messages on their video receiver's OSD
+* Pilots can see lap times and splits on their video receiver's OSD
+
+**TODO**: Insert demo video
+
+## Software Setup in RotorHazard
+
+1. In your server config file, edit VRX_SERVER "ENABLED" to true
+   
+   * 
+	```json
+		"VRX_SERVER": {
+				"HOST": "localhost",
+				"ENABLED": true
+			},	
+	```	
+   * The default config file is located at `RotorHazard/src/server/config.json`,
+1. Install the ClearView Receviers  
```

Comment:
Misspelling.  Should this say "... libraries"?

Reply:
Yes